app/server/endpoint/src/main/java/io/syndesis/server/endpoint/v1/handler/connection/ConnectorIconHandler.java

```diff
@@ -85,9 +85,6 @@ public Connector create(MultipartFormDataInput dataInput) {
         try {
             InputPart filePart = dataInput.getParts().iterator().next();
             try (InputStream result = filePart.getBody(InputStream.class, null)) {
-                if (result == null) {
```

Comment:
This is a false positive, result can be `null` here. See https://github.com/spotbugs/spotbugs/issues/259

Reply:
Ooops!